variables.tf

```diff
@@ -80,8 +80,14 @@ variable "load_balancer_update_timeout" {
   default     = "10m"
 }
 
+variable "log_enable" {
+  description = "Enable logging"
+  default     = false
```

Comment:
I would prefer to set the default to `true` to keep backward compatiblity for people who are using `master` branch of this module.

Reply:
ok, changed